packages/components/src/types/props/label.ts

```diff
@@ -104,6 +114,21 @@ export const validateLabel = (component: Generic.Element.Component, value?: Labe
 	watchValidator(component, '_label', (value) => typeof value === 'string', LABEL_VALUES, value, getValidationOptions(options));
 };
 
+// export const validateLabelWithMarkdown = (component: Generic.Element.Component, value?: LabelWithMarkdownPropType, options: WatchStringOptions = {}): void => {
```

Comment:
Der Block kann vermutlich ganz raus, da jetzt in `validateLabelWithExpertSlot` enthalten?

Reply:
Der Expert-Slot wird auf `String` umgestellt. Dann wird die validateLabelMarkdown wieder verwendet.